src/bp/ui-admin/src/Pages/Workspace/Bots/index.tsx

```diff
@@ -82,6 +82,7 @@ class Bots extends Component<Props> {
       this.props.fetchLicensing()
     }
 
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
     telemetry.startFallback(api.getSecured()).catch()
```

Comment:
I think you can simply make `componentDidMount` async and then await this call, or am I missing something?

Reply:
Yeah could be, just went the easy way :P